lib/analysis/modules/ies_enkf.c

```diff
@@ -199,9 +193,9 @@ void ies_enkf_updateA( void * module_data,
    ies_enkf_data_type * data = ies_enkf_data_safe_cast( module_data );
    const ies_enkf_config_type * ies_config = ies_enkf_data_get_config( data );
 
+
    int nrobs_msk     =ies_enkf_data_get_obs_mask_size(data); // Total number of observations
```

Comment:
In which way `nrobs_msk` is used? since we use `nrobs_inp` as the number of active observations.

Reply:
nrobs_msk is only used for diagnostics while printing the number of active observations versus the total number of observations.